src/Button/Button.tsx

```diff
@@ -123,6 +123,7 @@ export class Button extends React.PureComponent<Props> {
       // use <button>s for all disabled links and things with no href prop (buttons)
       return (
         <button
+          aria-label={typeof value === "string" ? (value as string) : null}
           {...additionalProps}
```

Comment:
Should we put other props before this as well (other than `classes`)?

Reply:
I thought about it but I figured it could mess with some things and just wanted to avoid that risk. For example the type prop on the dewey Button accepts linkPlain, primary, secondary, etc., but here in the component when returning an html button type represents the html button type field and not the dewey Button type field

Reply:
I'm wondering if we should just add a new optional prop `ariaLabel`. Something like:
```
const { ... ariaLabel ... } = this.props;
...
const aria = ariaLabel ?? typeof value === "string" ? (value as string) : null;
...
<Button
  {...additionalProps}
  ..
  aria-label={aria}
/>


```

This way we can keep the pattern where we spread `additionalProps` on top. What do you think?

Reply:
I think I could go either way - Do you feel strongly about it?

Reply:
It seems a bit cleaner imo

Reply:
Yeah sounds good 👍